docs/presentation/index.html

```diff
@@ -84,24 +90,24 @@
 
           <section>
             <h2>Light Metrics</h2>
-            <p class="fragment">intensity</p>
-            <p class="fragment">wavelength</p>
-            <p class="fragment">correlated color temperature</p>
+            <p class="fragment">Intensity</p>
+            <p class="fragment">Wavelength</p>
+            <p class="fragment">Correlated Color Temperature</p>
+            
             <img class="fragment" src="img/light-charts.png" />
 
             <aside class="notes">
-              Light emits photons onto plants which causes photosynthesis. Photons have wavelengths which are measured in nanometers. Some wavelengths are more desirable than other such and red and blue. There is also an intensity for each wavelength that can be measured.
+              Light emits photons onto plants which causes photosynthesis. Photons have wavelengths which are measured in nanometers. Some wavelengths are more desirable than other such and red and blue. There is also an intensity for each wavelength that can be measured. Additionally, white light can vary in how warm or cool the light appears. The color temperature of the white light can also dramatically affect plant growth.
             </aside>
           </section>
 
           <section>
             <h2>Growth Stages</h2>
-
-            <p class="fragment">germination</p>
-            <p class="fragment">seedling</p>
-            <p class="fragment">vegetative</p>
-            <p class="fragment">flowering</p>
-
+            <p class="fragment">Germination</p>
+            <p class="fragment">Seedling</p>
+            <p class="fragment">Vegetative</p>
+            <p class="fragment">Flowering</p>
+            <p class="fragment">Anything that is important to characterize</p>
```

Comment:
My thought was that stages should be rigidly defined, meaning you can only pick from a pre-selected list of stages for the recipe to be valid. This will help develop a more uniform pattern for parsers to use. That being said, we have not come up with what the list of available stages there are. The list of potential candidates are in this survey https://docs.google.com/forms/d/e/1FAIpQLSe6h1lA1nBl3xyMpp6_RMoFksJ_gKzOrvybXDyGu1hWxPirZw/viewform

which are:
- Sowing
- Germination
- Seedling
- Juvenile
- Adult
- Vegetative
- Flowering
- Dormant

Some of these stage names could overlap with each other like sowing and germination.

I think we should remove `Anything that is important to characterize` until we come up with a more concise idea about the growth stages for the standard.

Reply:
That sounds good. I didn't see the same stages listed in the general Growth Recipe Standard Doc, but I'm cool with only listing these here. We should probably list the same in the presentation, and also make sure that in the doc, we've also got those stages written down.